### ISSUE TITLE
Item popup title missing ARIA heading role

### ISSUE THREAD

**Issue by kirsty-hames** [cgkineo/adapt-hotgrid#120]
### Subject of the issue
The popup item title is read by a screen reader but it isn't announced as a 'heading' neither is the heading `aria-level` announced.

### Your environment
* 5.28.15 FW / 4.3.5 Hotgrid
* Mac Safari running voiceOver

### Steps to reproduce
With a screen reader running, select a Hotgrid item to trigger the popup. Note the item title is readable but it isn't announced as a 'heading' neither is the heading `aria-level`.

Note, same issue raised for Hotgraphic https://github.com/adaptlearning/adapt-contrib-hotgraphic/issues/265
